lib/data_fetchers/git_fetcher.rb

```diff
@@ -38,18 +38,31 @@ def line_changes
   end
 
   def languages
-    languages = github.languages.map do |language, percent|
+    languages = github.languages
+      .merge(gitlab.languages) { |key, value1, value2| ((value1 + value2) / 2.0).round }
```

Comment:
My understanding of this is that we use the average of the percentages of gitlab and github. This may be misleading as one of the hosters might have many more lines/projects than the other. If we mirror some big projects from github to gitlab it will also have quite some influence.

Not sure how to fix this, but we might:
* show both numbers seperately
* use some weights based on projects/line numbers (not sure what the statistic is based on)
* use linguist on checkouts of both github and gitlab projects instead of using gitlabs stats

The influence of having the github projects mirrored on gitlab could be quite big, but I guess we need to do some ballpark calculations to estimate this.